docs/modules/certification.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Application Community Certification
+nav_order: 1
+has_children: false
+---
+
+# Application Community Certification
+
+## From ideas to applications
+
+Before creating an application for NethServer 8, consider if it's really
+necessary. Sometimes, a simple documentation page is sufficient and
+requires much less effort. When trying out new ideas, make sure to
+document every step and comment in a [community discussion under the
+Feature category](http://community.nethserver.org/c/feature).
+
+If the idea gains traction, then it's time to think about a new
+application. When creating one, ensure the following requirements are met:
+
```

Comment:
Add a paragraph to check that the application is compliant with the laws of the countries where it will be used, or at least if and how it complies with local laws. Eg. GDPR, NIS2, DORA, SOX